plugins/jenkins.js

```diff
@@ -294,6 +493,122 @@ function executePlugin (channel, callback, text) {
     });
 }
 
+function updateNulls (msgCB) {
+    var nullResults = [];
+
+    // retrieve builds with null results from jenkinsStore
+    _.forEach(this.dataStore, function (value, key) {
+        if (value[result] !== 'SUCCESS' && value[result] !== 'FAILURE') {
+            nullResults.push({ "name": key, "url": value[url] });
+        }
+    });
+
+    if (nullResults.length > 0) {
+        nullResults.forEach(function(build) {
+            makeRequest( { url: build.url + 'api/xml' }, function (data) {
+
+            });
+        });
+    }
+}
+
+function makeRequest (object, callback, responseCB, form) {
```

Comment:
Once we've got everything merged up, let's go back and clean out extra junk like this copy of `makeRequest`.